payments_service/braintree/forms.py

```diff
@@ -21,6 +27,12 @@ class SubscriptionForm(forms.Form):
     # things like recurring donations. Solitude validates this value to make
     # sure you can't adjust fixed price subscriptions.
     amount = forms.DecimalField(required=False)
+    email = forms.EmailField(required=False, max_length=255)
```

Comment:
Max length of an email address should be 254 see http://www.rfc-editor.org/errata_search.php?rfc=3696&eid=1690